solhint.js

```diff
@@ -5,6 +5,19 @@ const linter = require('./lib/index');
 const _ = require('lodash');
 const fs = require('fs');
 const process = require('process');
+const formatterOptions = [
+    'checkstyle',
+    'codeframe',
+    'compact',
+    'html',
+    'jslint-xml',
+    'json',
+    'junit',
+    'stylish',
+    'table',
+    'tap',
+    'unix',
+    'visualstudio'];
```

Comment:
Since we delegate the formatting to eslint, instead of hardcoding the list of possible formatters I think a better option is to handle this in the `catch` block when the formatter is required.

The downside of this approach is that the error will be throw after the linter is executed, and that wastes user time unnecessarily. But I think it's OK for now. (A possible improvement to avoid this is to get the formatter before the linter is run.)